### PR TITLE
Feature/seeders

### PR DESCRIPTION
**Pull request by winfred-agbotey**

### CODE REVIEW

File: compose.yaml [hunk 1]
```diff
@@ -1,9 +1,16 @@
 services:
   postgres:
+    container_name: mentorship
     image: 'postgres:latest'
     environment:
-      - 'POSTGRES_DB=mydatabase'
-      - 'POSTGRES_PASSWORD=secret'
-      - 'POSTGRES_USER=myuser'
+      POSTGRES_DB: mentorship-tracker
+      POSTGRES_PASSWORD: mawuli302
```
Comment (daniel-bontii) [inline compose.yaml]: Hide this in a .env file

File: src/main/java/com/mentorshiptracker/models/Role.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "roles")
```
Comment (daniel-bontii) [inline src/main/java/com/mentorshiptracker/models/Role.java:15]: Add an equal and hash

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+server:
+  port: 5000
+  error:
+    include-binding-errors: always
+    include-message: always
+
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5222/mentorship-tracker
+    username: mawuli
```
Comment (daniel-bontii) [inline src/main/resources/application.yml]: Hide the username and password in the environment

**Review by daniel-bontii — CHANGES_REQUESTED** [review]
Hide the database credentials for obvious security reasons

File: compose.yaml (hunk 1 above)
Comment (winfred-agbotey) [reply compose.yaml]: fixed

File: src/main/resources/application.yml (hunk 3 above)
Comment (winfred-agbotey) [reply src/main/resources/application.yml]: fixed

File: .env [hunk 4]
```diff
@@ -0,0 +1,3 @@
+POSTGRES_PASSWORD=mawuli302
```
Comment (daniel-bontii) [inline .env]: Add this file to the .gitIgnore. We did this to hide these credentials.
Add a .env.example instead

**Review by daniel-bontii — CHANGES_REQUESTED** [review]
Make sure to truly hide the credentials

File: .env (hunk 4 above)
Comment (winfred-agbotey) [reply .env]: done

**Pull request merged**